### PR TITLE
fix: Fetch only limit orders

### PR DESCRIPTION
**Pull request by holzeis**
I am surprised this didn't had a bigger impact, but apparently we where fetching also market orders.

### CODE REVIEW

File: coordinator/src/orderbook/db/orders.rs [hunk 1]
```diff
@@ -181,24 +181,12 @@ impl From<OrderbookNewOrder> for NewOrder {
     }
 }
 
-pub fn all(
-    conn: &mut PgConnection,
-    show_expired: bool,
-    show_failed: bool,
```
Comment (luckysori) [inline coordinator/src/orderbook/db/orders.rs:187]: ❓ Why did you remove this?

I have no opinion on it, but maybe this was useful to someone? Cc @bonomat.

Comment (holzeis) [reply coordinator/src/orderbook/db/orders.rs:187]: because it was unused.

Comment (luckysori) [reply coordinator/src/orderbook/db/orders.rs:187]: It was part of the HTTP API of the coordinator. In theory someone can query this endpoint to get information about the orderbook.

Comment (holzeis) [reply coordinator/src/orderbook/db/orders.rs:187]: Hmm.. Unless we were using it for some admin task (in which case that api should be in the admin file) it is not used by either the app nor the maker. (Unless I am missing something.

Happy to add it back if we think it's needed. 🙂

Comment (holzeis) [reply coordinator/src/orderbook/db/orders.rs:187]: Note, even if it is used for an admin task, I think it would be better to connect to the database directly for this query. (instead of exposing it publicly)

Comment (luckysori) [reply coordinator/src/orderbook/db/orders.rs:187]: > I think it would be better to connect to the database directly for this query.

That's a good point! Let's remove it then.

Comment (bonomat) [reply coordinator/src/orderbook/db/orders.rs:187]: No hard feelings on this :)
